pkg/runtime/authorizer/authorizer.go

```diff
@@ -0,0 +1,113 @@
+/*
```

Comment:
Can unit tests be added to test the functionality here?

Reply:
Done :+1: